### PR TITLE
[ERDE-2821] Update API Clients

### PR DESCRIPTION
**Pull request by sophos-rickc**
## This PR will:
- [x] *Update the endpoint URLs for `api` and `authApi`.*
- [x] *Upgrade all `npm` packages to most recent stable version(s).*
- [x] *Remove @sophos-daniels from the GitHub workflows

## Checklist
- [x] Updates dependency versions
- [x] All new and existing tests pass
- [x] Affected documentation (internal/external) updated accordingly

## Merge considerations
This PR should be merged (and, thus, deployed) after `erde` [PR#1214](https://github.com/sophos-internal/erde/pull/1214) is deployed to `us-west-2`

### CODE REVIEW

File: src/generated/api/.openapi-generator/FILES [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .gitignore
 .npmignore
+.openapi-generator-ignore
```
Comment (sophos-carlos) [inline src/generated/api/.openapi-generator/FILES:3]: Is this file something that's generated? If it was a "rename" manually, no need to store it here.

File: src/generated/authApi/.openapi-generator/FILES [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .gitignore
 .npmignore
+.openapi-generator-ignore
```
Comment (sophos-carlos) [inline src/generated/authApi/.openapi-generator/FILES:3]: See prev comment

File: src/generated/api/.openapi-generator/FILES (hunk 1 above)
Comment (sophos-rickc) [reply src/generated/api/.openapi-generator/FILES:3]: Note that the path is `src/generated/api/...`. Yes, this is a generated file

File: src/generated/authApi/.openapi-generator/FILES (hunk 2 above)
Comment (sophos-rickc) [reply src/generated/authApi/.openapi-generator/FILES:3]: See prev response...

**Review by sophos-carlos — COMMENTED** [review]
Looking good, a tiny comment. Have not gone through the lock file.

**Comment by sophos-rickc** [comment]
> Looking good, a tiny comment. Have not gone through the lock file.

Those comments were on generated files, @sophos-carlos. Would you, please, take another look and approve?  Thanks!

**Review by sophos-carlos — APPROVED** [review]
Looks good

**Pull request merged**